gcp/gcp.go

```diff
@@ -75,6 +76,29 @@ func DefaultCredentials(ctx context.Context) (*google.Credentials, error) {
 	return adc, nil
 }
 
+// FakeDefaultCredentials gets GCP credentials from a fake JSON credentials
+// file in memory. This function is meant for use as a helper in example code,
+// to be replaced by gcp.DefaultCredentials(ctx) or google.CredentialsFromJSON
+// in non-example code.
+// See https://cloud.google.com/docs/authentication/production
```

Comment:
Don't think this reference is useful here.

Reply:
Removed it.